index.html

```diff
@@ -1,7 +1,15 @@
 <!DOCTYPE HTML>
-<html lang="en">
+<html lang="en" class="no-js">
 <head>
     <meta http-equiv="content-type" content="text/html; charset=utf-8">
+	<script>
+	  document.documentElement.classList.remove("no-js");
+	</script>
+    <script>
+    /*! modernizr 3.6.0 (Custom Build) | MIT *
```

Comment:
Please do like so:

```
/*-
* The modernizr script is here to make sure that webp images are shown, where possible, instead of 
* png images.
* It works basically like this: 
*/
```

Reply:
Checking browser support for webp (basic, lossless, etc) and based on that, sets a css class on the top-level html tag.